web-registry/src/components/organisms/ProjectTopSection/ProjectTopSection.tsx

```diff
@@ -33,7 +32,7 @@ function ProjectTopSection({
   isGISFile,
   batchData,
 }: {
-  data?: ProjectByHandleQuery;
+  data?: any; // TODO: when all project are onchain, this can be ProjectByOnChainIdQuery
```

Comment:
i notice we have a few todo comments related to when projects go on-chain, just wanted to see if we had an issue open where we could perhaps reference these todo's to keep track